xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.h

```diff
@@ -63,6 +63,8 @@ class CSinkControlProtocol : public Protocol
     VOLUME,
     FLUSH,
     TIMEOUT,
+    SILENCECOUNTERUPDATED,
```

Comment:
As you use it as a tristate in the sink. Once for using an infty value, then to set it to zero. This is just used to transmit the setting into the sink.